libs/datamodel/core/src/transform/dml_to_ast/generator_serializer.rs

```diff
@@ -0,0 +1,58 @@
+use crate::{ast, configuration::Generator};
+
+pub struct GeneratorSerializer {}
+
+impl GeneratorSerializer {
+    pub fn add_generators_to_ast(generators: &[Generator], ast_datamodel: &mut ast::SchemaAst) {
+        let mut tops: Vec<ast::Top> = Vec::new();
```

Comment:
nitpick: this could be `Vec::with_capacity(generators.len() + ast_models.tops.len())`;